pkgs/development/python-modules/pytest-dependency/default.nix

```diff
@@ -9,7 +9,16 @@ buildPythonPackage rec {
     sha256 = "bda0ef48e6a44c091399b12ab4a7e580d2dd8294c222b301f88d7d57f47ba142";
   };
 
-  propagatedBuildInputs = [ pytest ];
+  nativeBuildInputs = [ pytest ];
```

Comment:
If you do this it doesn't add `pytest` to `result/nix-support/propagated-build-inputs`.
Since it's in the `install_requires` of pytest-dependency I think that is desirable.

What I'd suggest, is to add `pytest` to `propagatedBuildInputs` and `checkInputs`.

Reply:
you're right, fixed. thanks!